easybuild/easyconfigs/w/WIEN2k/WIEN2k-21.1-intel-2021a.eb

```diff
@@ -28,7 +33,7 @@ dependencies = [
 osdependencies = [('glibc-devel', 'libc6-dev')]  # required for libpthread.a
 
 # remote = 'pbsssh'
-wien_mpirun = 'mpirun  -np _NP_ _EXEC_'
+wien_mpirun = 'srun  -n_NP_ _EXEC_'
```

Comment:
@akesandgren Is this an intended change?

Reply:
oops no...